autoware_map_msgs/srv/GetSelectedLanelet2Map.srv

```diff
@@ -0,0 +1,9 @@
+# Osm file ids which are selected to be loaded
+string[] cell_ids
+
+---
+# Header
+std_msgs/Header header
+
+# Newly loaded Lanelet map
+autoware_map_msgs/LaneletMapBin differential_map
```

Comment:
For PointCloud, the map is provided divided by cells.
https://github.com/autowarefoundation/autoware_msgs/blob/54e077bbc51af5ea4efcfc8b6183c85968d3fbb0/autoware_map_msgs/srv/GetSelectedPointCloudMap.srv#L9

Is it more convenient for the Lanelet map to be integrated into a single map?

Reply:
Unlike PointCloudMap, I think it is better to to combine them on the service side, not on the client side.
We have to decode the received message in order to access the actual data so I think it is easier for the users to just decode once rather than decoding individual cells and merge them together.

Reply:
OK. I understood.
@StepTurtle Then could you please just modify the variable name to `differential_map`?

Reply:
Sorry @YamatoAndo, Isn't the variable name `differential_map` anyway? Did I miss something?

Did you mean something like `new_lanelet2_cells`?

Reply:
@StepTurtle 
> Did you mean something like new_lanelet2_cells?
Yes. The name `differential_map` is not suitable since it doesn't differential load, so I request that it be changed to something else.

I think `new_lanelet2_cells` would be appropriate. `map`, `lanelet2_map`, or `lanelet2_cells` would also be acceptable.

Reply:
OK, `lanelet2_cells` looks good, but if we want to unify with point cloud I can put `new_` at the beginning.

Reply:
changed with `lanelet2_cells`